integration/docker/test_basic.py

```diff
@@ -183,6 +183,36 @@ def test_recurring_snapshot_live_update_retain(clients):  # NOQA
     assert len(left) == 0
 
 
+def test_bgtasks(clients):  # NOQA
+    for host_id, client in clients.iteritems():
+        break
+
+    setting = client.by_id_setting("backupTarget")
+    setting = client.update(setting, value=common.get_backupstore_url())
+
+    volume = client.create_volume(name=VOLUME_NAME, size=SIZE,
+                                  numberOfReplicas=2)
+    volume = volume.attach(hostId=host_id)
+
+    num_tasks = 3
+    snaps = map(lambda _: volume.snapshotCreate(), range(num_tasks))
+    for snap in snaps:
+        volume.snapshotBackup(name=snap["name"])
+
+    tasks0 = volume.bgTaskQueue()
+    assert len(tasks0) == num_tasks
+    assert tasks0[0]["num"] == 1
+    assert tasks0[0]["finished"] == ""
+
+    time.sleep(60)
+
+    tasks1 = volume.bgTaskQueue()
+    assert len(tasks1) == 1
```

Comment:
Why there will be only one task after 60 seconds, and it's still not finished? Not quite understand the logic here..